modules/exploits/unix/webapp/wp_crop_rce.rb

```diff
@@ -0,0 +1,397 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+  super(update_info(
+    info,
+    'Name'            => 'WordPress Crop-image Shell Upload',
+    'Description'     => %q{
+        This module exploit a path traversal and a local file inclusion
+        vulnerability on WordPress versions 4.9.8 and less.
+        The crop-image function allow an user, with at least author privileges,
+        to resize an image an perform a path traversal by changing the _wp_attached_file
+        reference during the upload. The second part of the exploit will include
+        this image in the current theme by changing the _wp_page_template attribute
+        when creating a post.
+    },
+    'License'         => MSF_LICENSE,
+    'Author'          =>
+    [
+      'RIPSTECH Technology', # Discovery
+      'Wilfried Becard'    # Metasploit module
+    ],
```

Comment:
Please add any more references you may have come across while writing your module. 

```suggestion
    ],
  'References'      =>
    [
      [ 'CVE', '2019-8942' ],
      [ 'CVE', '2019-8943' ],
      [ 'URL', 'https://blog.ripstech.com/2019/wordpress-image-remote-code-execution/']
    ],
```